src/autoscaler/metricscollector/exampleconfig/example.yml

```diff
@@ -1,8 +1,11 @@
 cf:
-  api: "https://api.example.com"
-  grant_type: "client_credentials"
-  client_id: "cf_id"
-  secret: "cf_secret"
+  api: "https://api.bosh-lite.com"
+  grant_type: "password"
+  username: "admin"
+  password: "admin"
+  # grant_type: "client_credentials"
```

Comment:
This is a sample config, I commented these lines just to show client_credential grant can be used as well. 

I will remove mc.yml, and just leave example.yml there.